docs/DQC_IFRS_0092/DQC_0092.md

```diff
@@ -1,23 +1,82 @@
 
 ## Problem solved by the rule
 The intent of the rule is to identify those facts that have been entered with a positive value when the value in fact should have been a negative amount.
 
+## Rule description 
+
+### Part 1
+
+Let **non-positive excluded axis/member pairs** be the following set of axis/member combinations:
+
+* CarryingAmountAccumulatedDepreciationAmortisationAndImpairmentAndGrossCarryingAmountAxis
+    * AccumulatedImpairmentMember
+    * AccumulatedDepreciationAmortisationAndImpairmentMember
+    * AccumulatedDepreciationAndAmortisationMember
+* RetrospectiveApplicationAndRetrospectiveRestatementAxis
+    * IncreaseDecreaseDueToChangesInAccountingPolicyAndCorrectionsOfPriorPeriodErrorsMember
+    * FinancialEffectOfChangesInAccountingPolicyMember
+    * IncreaseDecreaseDueToChangesInAccountingPolicyRequiredByIFRSsMember
+    * IncreaseDecreaseDueToVoluntaryChangesInAccountingPolicyMember
+    * FinancialEffectOfCorrectionsOfAccountingErrorsMember
+* FinancialEffectOfTransitionFromPreviousGAAPToIFRSsAxis
+    * EffectOfTransitionToIFRSsMember
+* RedesignationAxis
+    * RedesignatedAmountMember
+* FairValueAsDeemedCostAxis
+    * AggregateAdjustmentToCarryingValueReportedUnderPreviousGAAPMember
+* DepartureFromRequirementOfIFRSAxis,
+    * IncreaseDecreaseDueToDepartureFromRequirementOfIFRSMember
+* ComponentsOfEquityAxis
+    * TreasurySharesMember
+* TypesOfRisksAxis
+    * RiskDiversificationEffectMember
+* AmountsArisingFromInsuranceContractsAxis
+    * ReinsurersShareOfAmountArisingFromInsuranceContractsMember
+* NetDefinedBenefitLiabilityAssetAxis
+    * PresentValueOfDefinedBenefitObligationMember
+    * PlanAssetsMember
+    * EffectOfAssetCeilingMember
+
+Let **non-positive sub-string exclusions** be the following set of strings:
+
+* adjust
+* consolidat
+* netting
+* reconcili
+* reclass
+* change
+* increasedecreas
+
+Let **excluded axes** be as defined in DQC0080.
+
+For each fact with a concept in the list of **non-negative concepts** defined in DQC.IFRS.0080, and a value of EliminationOfIntersegmentAmountsMember for the SegmentConsolidationItemsAxis, the value MUST be less than or equal to zero unless:
+
+* the fact has a dimension with a non-default value that is in the set of **non-positive excluded axis/member pairs**; or
+* the fact has a dimension and member that is in the set of **excluded axes**; or 
+* any string in **non-positive sub-string exclusions** is a case-insensitive substring of the local name of any non-default dimension member applied to the fact.
+
+### Part 2
+
+Let **asset-related concepts** be the list of concepts defined in [dqc_0091_ifrs_2019_concepts.csv](https://github.com/DataQualityCommittee/dqc_us_rules/blob/master/dqc_us_rules/resources/DQC_IFRS_0091/dqc_0091_ifrs_2019_concepts.csv).
```

Comment:
This is referencing a 2019 list of concepts.  Does this need to depend on the IFRS taxonomy version?

Should this list be maintained in the DQR repository?